### ISSUE TITLE
[FEATURE REQUEST] A Docker image, and a helm chart that uses this image

### ISSUE THREAD

**Issue by hakan-77** [apache/polaris#365]
### Is your feature request related to a problem? Please describe.

I located a helm chart in the repo, but 
1) it's not packaged and deployed 
2) uses an image from a local registry of localhost:5001/polaris (??)

### Describe the solution you'd like

Have a helm chart and a docker image.

### Describe alternatives you've considered

We already use Nessie (through the official helm chart) and wanted to give polaris a shot.

**Comment by czy006** [comment]
I'm interested in it, and I think I can do it. @hakan-77

**Comment by hakan-77** [comment]
@czy006 just saw this: https://github.com/apache/polaris/issues/152

**Comment by MonkeyCanCode** [comment]
Yeah, it is not hard to publish this. However, i haven't done so with Apache project. I am assuming we will need to use some specialize account then update CI pipeline to package those and update?

**Comment by enesyesil** [comment]
Hey, Is this still open? Happy to take it on.

**Comment by dimas-b** [comment]
Please try the latest helm charts from https://polaris.apache.org/downloads/

I hope this issue has been addressed.
